yarn.lock

```diff
@@ -0,0 +1,4 @@
+# THIS IS AN AUTOGENERATED FILE. DO NOT EDIT THIS FILE DIRECTLY.
+# yarn lockfile v1
```

Comment:
We don't need this

Reply:
oh sorry that might have been build when I was trying to build jager-ui 
I will remove this

Reply:
please also remove bin/golangci-lint